tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -352,110 +274,112 @@ def delete_device(self):
-        # PROTECTED REGION ID(CentralNode.telescopeHealthState_read) ENABLED START #
-        '''
-        if ((self._subarray1_health_state == 0) & (self._subarray2_health_state == 0)):
-            self._telescope_health = 0
-        elif ((self._subarray1_health_state == 2) & (self._subarray2_health_state == 2)):
-            self._telescope_health = 2
-        elif ((self._subarray1_health_state == 1) | (self._subarray2_health_state == 1)):
-            self._telescope_health = 1
-        else:
-            self._telescope_health = 3
-
-        '''
+        # PROTECTED REGION ID(CentralNode.telescope_healthstate_read) ENABLED START #
+        """ This method reads the Telescope health state."""
         return self._telescope_health_state
-        # PROTECTED REGION END #    //  CentralNode.telescopeHealthState_read
+        # PROTECTED REGION END #    //  CentralNode.telescope_healthstate_read
 
     def read_subarray1HealthState(self):
-        # PROTECTED REGION ID(CentralNode.subarray1HealthState_read) ENABLED START #
+        # PROTECTED REGION ID(CentralNode.subarray1_healthstate_read) ENABLED START #
+        """ This method reads the subarray 1 health state"""
         return self._subarray1_health_state
-        # PROTECTED REGION END #    //  CentralNode.subarray1HealthState_read
+        # PROTECTED REGION END #    //  CentralNode.subarray1_healthstate_read
 
     def read_subarray2HealthState(self):
-        # PROTECTED REGION ID(CentralNode.subarray2HealthState_read) ENABLED START #
+        # PROTECTED REGION ID(CentralNode.subarray2_healthstate_read) ENABLED START #
+        """ This method reads the subarray 2 health state"""
         return self._subarray2_health_state
-        # PROTECTED REGION END #    //  CentralNode.subarray2HealthState_read
+        # PROTECTED REGION END #    //  CentralNode.subarray2_healthstate_read
 
     def read_activityMessage(self):
-        # PROTECTED REGION ID(CentralNode.activityMessage_read) ENABLED START #
+        # PROTECTED REGION ID(CentralNode.activity_message_read) ENABLED START #
+        """ This method reads activity message """
         return self._read_activity_message
-        # PROTECTED REGION END #    //  CentralNode.activityMessage_read
+        # PROTECTED REGION END #    //  CentralNode.activity_message_read
 
     def write_activityMessage(self, value):
-        # PROTECTED REGION ID(CentralNode.activityMessage_write) ENABLED START #
+        # PROTECTED REGION ID(CentralNode.activity_message_write) ENABLED START #
+        """ This method reads activity message intended to write on the GUI"""
         self._read_activity_message = value
-        # PROTECTED REGION END #    //  CentralNode.activityMessage_write
+        # PROTECTED REGION END #    //  CentralNode.activity_message_write
 
 
     # --------
     # Commands
     # --------
 
     @command(
-    dtype_in=('str',), 
-    doc_in="List of Receptors to be stowed", 
+        dtype_in=('str',),
```

Comment:
Indentation?

Reply:
Nope, it is fine in the code. Resolving this comment.